includes/class-wc-siftscience-order-status.php

```diff
@@ -44,7 +53,9 @@ public function __construct( WC_SiftScience_Options $options ) {
 		public function get_status_options() {
 			$result = array( 'none' => 'Do Nothing' );
 			foreach ( wc_get_order_statuses() as $key => $val ) {
-				$result[ $key ] = $val;
+				// Remove the "wc-" prefix on status names because they're not elsewhere.
+				$k            = strpos( $key, 'wc-' ) === 0 ? substr( $key, 3 ) : $key;
```

Comment:
strpos( $key, 'wc-' ) === 0
it's better using Yoda condition  0 === strpos( $key, 'wc-' )